src/test/resources/htsjdk/samtools/util/sliver.sam

```diff
@@ -0,0 +1,3485 @@
+@HD	VN:1.5	GO:query	SO:coordinate
+@SQ	SN:chr1	LN:248956422	AS:38	M5:6aef897c3d6ff0c78aff06ac189178dd	UR:/seq/references/Homo_sapiens_assembly38/v0/Homo_sapiens_assembly38.fasta	SP:Homo sapiens
```

Comment:
Can you remove everything other than chr1 here?  Make for a smaller test input file.